ci/requirements-py27-cdat+pynio.yml

```diff
@@ -1,6 +1,8 @@
 name: test_env
 channels:
-  - ajdawson
+  - ajdawson  # cdat
+  - dbrown  # pynio
+  - asmeurer  # gcc-4.8.5, needed for pynio
```

Comment:
It looks like this may be in the main channel now:
```
$ conda search gcc
Fetching package metadata: ......
gcc                          4.8.2                         5  defaults
                             4.8.5                         3  defaults
libgcc                       4.8.4                         1  defaults
                          .  4.8.5                         1  defaults
```